package/libsolv.spec.in

```diff
@@ -295,34 +266,26 @@ export NO_BRP_STRIP_DEBUG=true
 %{_mandir}/man3/*
 
 %files demo
-%defattr(-,root,root)
 %{_bindir}/solv
 
-%if %{with perl_binding}
+%if %{with perl}
 %files -n perl-solv
-%defattr(-,root,root)
 %{perl_vendorarch}/*
 %endif
 
-%if %{with ruby_binding}
+%if %{with ruby}
 %files -n ruby-solv
-%defattr(-,root,root)
-%{ruby_vendorarch}/*
+%{rb_vendorlibdir}/*
 %endif
 
-%if %{with python_binding}
+%if %{with python2}
 %files -n python-solv
```

Comment:
Change this to `python2-solv`

Reply:
Why, just to do provides/obsoletes since all the depending packages say python-solv...